### ISSUE TITLE
limit commands

### ISSUE THREAD

**Issue by lupa18** [botgram/shell-bot#24]
is there any possibility to limit commands that bot can run?

thanks in advance

**Comment by mildsunrise** [comment]
nope, sorry. you have to modify the code yourself, this isn't implemented for now
